tt_metal/impl/dispatch/command_queue_interface.hpp

```diff
@@ -414,6 +414,9 @@ class SystemMemoryManager {
             TT_FATAL(
                 dispatch_constants::get(core_type).max_prefetch_command_size() *
                     (dispatch_constants::get(core_type).prefetch_q_entries() + 2) <=
+                this->get_issue_queue_size(cq_id),
+                "Issue queue for cq_id {} has size of {} which is too small",
```

Comment:
What is the valid range? 
What should user do when they see this error message?